chainercv/transforms/image/pca_lighting.py

```diff
@@ -0,0 +1,47 @@
+import numpy
+
+
+def pca_lighting(img, sigma, eigen_value=None, eigen_vector=None):
+    """Alter the intensities of input image using PCA.
+
```

Comment:
This explanation sounded like PCA is computed every time this function is called.

For example,
"perturbate color channels of images in principle directions of the covariance matrix of color channels" 

This is a little bit wordy, so please take this just as a reference.
Maybe it is smart to steal words from the original paper.